cmd/soroban-rpc/main.go

```diff
@@ -46,8 +46,8 @@ func main() {
 					branch = ""
 				}
 				fmt.Printf("soroban-rpc %s (%s) %s\n", config.Version, config.CommitHash, branch)
-				fmt.Printf("stellar-xdr %s\n", goxdr.CommitHash)
 			}
+			fmt.Printf("stellar-xdr %s\n", goxdr.CommitHash)
```

Comment:
I am debating two things:
- Should we make this format closer to the `soroban-cli version` output?
- And/or should we denote this as a file hash (vs a git commit), with something like:
  - `stellar-xdr sha256:2f16687fdf6f4bcfb56805e2035f69997f4b34c4`

Reply:
running `stellar-core version`, I received:
```
Warning: running non-release version v19.10.0-59-g57cc8198 of stellar-core
Warning: soroban-env-host-curr is running a pre-release version 0.0.16
v19.10.0-59-g57cc8198
rust version: rustc 1.69.0 (84c898d65 2023-04-16)
soroban-env-host:
    curr:
        package version: 0.0.16
        git version: 0c0cae6fa22b751e7fd95d7ce7556ac6d7b7010e
        ledger protocol version: 20
        pre-release version: 37
        rs-stellar-xdr:
            package version: 0.0.16
            git version: 53e1a9cf2335aff29305c72deb6f075e78915dad
            base XDR git version: 2f16687fdf6f4bcfb56805e2035f69997f4b34c4
```
so, I think that we should figure out how we want to present these version uniformly across all our command line utilities.